libs9s/s9soptions.cpp

```diff
@@ -3302,6 +3304,40 @@ S9sOptions::s3region() const
     return retval;
 }
 
+/**
+ * \returns The argument for the --only-cloud option
+ */
+bool
+S9sOptions::onlyCloud() const
```

Comment:
```suggestion
S9sOptions::getOnlyCloud() const
```
It would be more clear.